db/migrate/20200922180455_add_uuid_and_family_to_physical_storages.rb

```diff
@@ -0,0 +1,6 @@
+class AddUuidAndFamilyToPhysicalStorages < ActiveRecord::Migration[5.2]
+  def change
+    add_column :physical_storages, :uuid, :string
```

Comment:
`PhysicalStorage` already has `:ems_ref` and a `uid_ems` columns

The `ems_ref` is typically the `id` of the object or some equivalent, and the `uid_ems` is typically a UUID if the provider has one, otherwise equal to the `ems_ref`

Unless you need both of the existing ems_ref and uid_ems columns for other properties, can you store the uuid in the uid_ems column?